### DISCUSSION TITLE
Is there any way to disable "all matches"?

### DISCUSSION THREAD

**Discussion by tomKPZ** [marlonrichert/zsh-autocomplete#265] [Q&A] [answered]
"All matches" will redundantly suggest what's already been suggested.  Ex:
```
$ ls
files
dev/   docs/  mnt/   net/   pics/  tmp/   vids/
all matches  (press Shift-Tab to insert)
dev docs mnt net pics tmp vids
```
Is there any way to avoid the same options cluttering the suggestions?

**Comment by marlonrichert** [comment]
It's not redundant _per sé_ (even though it might be to you personally): It shows you that you can press Shift-Tab to insert _all_ of the matches listed (instead of just one). 🙂

Anyway… No, hiding that is currently not possible. But feel free to [open a feature request](https://github.com/marlonrichert/zsh-autocomplete/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=) for it.

**Reply by tomKPZ** [reply]
Filed https://github.com/marlonrichert/zsh-autocomplete/issues/268

Thank you for being willing to accept new features and for your quick response :)

**Comment by marlonrichert** [comment, marked as answer]
I just discovered that this is, in fact, already possible through Zsh's completion system. The following hides both types of Shift-Tab completions, but without disabling them:
```zsh
zstyle ':completion:*:all-matches' hidden all
zstyle ':completion:*:unambiguous' hidden all
```

**Reply by tomKPZ** [reply]
That's exactly what I'm looking for!  Thanks!
